### PR TITLE
FindShortestNoVidPath

### PR DESCRIPTION
**Pull request by casualwind**
Add the class FindShortestNoVidPath.
The class means that FindShortestPath when there is one vid does not exist.

### CODE REVIEW

**Comment by casualwind** [comment]
@HarrisChu 79#PR was submitted before, but during the modification process, PR may be closed due to forced rollback on my computer. So I open the new PR. Thank you for your review.

**Comment by wey-gu** [comment]
@casualwind btw, we are setting up a special interest group and will launch a test program, and before the public publish, we would like to have an internal beta test, are you interested to join?

We will basically release some utils and procedures that we could easily generate some perf tests and report, which could be shared in the community.

我用中文说下哈，我们会搞一个测试的社区兴趣小组，小组里准备发起一个公开的社区活动：大家一起测试 NebulaGraph 的性能，会提供一些工具和手册，让大家在不同的环境里跑出来结果，分享给大家。我们准备在启动公开之前，开启内测。不知道你是否感兴趣哈？

**Comment by casualwind** [comment]
@wey-gu I am interested in getting involved in the community🤭

**Comment by wey-gu** [comment]
> @wey-gu I am interested in getting involved in the community🤭

Awesome! Would you mind mail me your wechat id via weyl.gu at gmail dot com?

We are about to have the internal discussion about this in this week :)

**Comment by casualwind** [comment]
> > @wey-gu I am interested in getting involved in the community🤭
> 
> Awesome! Would you mind mail me your wechat id via weyl.gu at gmail dot com?
> 
> We are about to have the internal discussion about this in this week :)

I have try to contacted with you via wechat.

File: nebula_bench/scenarios/find_path.py [hunk 1]
```diff
@@ -2,8 +2,16 @@
...
     csv_path = "social_network/dynamic/person_knows_person.csv"
     rank = 100
+
+class FindShortestPath(BaseFindShortestPath):
+    abstract = False
+    nGQL = "FIND SHORTEST PATH FROM {0} TO {1} OVER * YIELD path as p"
+
+class FindShortestNoVidPath(BaseFindShortestPath):
+    abstract = False
+    nGQL = "FIND SHORTEST PATH FROM {0} TO -1 OVER * YIELD path as p"
```
Comment (HarrisChu) [inline nebula_bench/scenarios/find_path.py:17]: could you add a newline in the end?
https://thoughtbot.com/blog/no-newline-at-end-of-file

---
在文件末尾加一个空行哈

Comment (casualwind) [reply nebula_bench/scenarios/find_path.py]: I have add the line at end
-------
加上了一个空行哈

**Review by HarrisChu — APPROVED** [review]

**Pull request merged**